woocommerce/woocommerce
# Fix CES style for WP 6.7

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #52369  .

This PR fixes CES modal style issue with WP 6.7. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch. Check WordPress Beta Tester under `Plugins` section.
2. Go through the core profiler (do not skip, but you don't have to connect to Jetpack)
3. Navigate to `Tools -> Beta Testing`
4. Choose `Bleeding edge` and `Beta/RC Only` then save the changes.
5. Navigate to `Dashboard -> Updates`
6. Click on `Update to version 6.7-RCx`
7. Navigate to `Products -> Add New`
8. Fill the data and click on `Publish`
9. Click on `Give feedback` and you should see the CES modal.

Note: I've noticed that I need to click `Give feedback` twice. This seems to be an existing bug, and I'll address it in a separate PR.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Fix broken CES style for WP 6.7

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
